packages/design/src/LabelInput/LabelInput.tsx

```diff
@@ -14,32 +14,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import styled from 'styled-components';
-import PropTypes from 'prop-types';
+import styled, { fontSize, margin, padding } from 'design/styled';
 
-import { space } from 'design/system';
+export interface LabelInputProps {
+  hasError?: boolean;
+}
 
-const LabelInput = styled.label`
+export const LabelInput = styled.label([
+  fontSize,
+  margin,
+  padding,
+])<LabelInputProps>`
   color: ${props =>
     props.hasError ? props.theme.colors.error.main : props.theme.colors.light};
   display: block;
   font-size: 11px;
   font-weight: 500;
   text-transform: uppercase;
   width: 100%;
-  ${space}
 `;
 
-LabelInput.propTypes = {
-  hasError: PropTypes.bool,
-};
-
 LabelInput.defaultProps = {
-  hasError: false,
   fontSize: 0,
```

Comment:
I noticed the `font-size` for labels has changed from 11px to 10px, removing this line should fix it.